specs-go/config.go

```diff
@@ -22,11 +22,36 @@ type Spec struct {
 	Hooks Hooks `json:"hooks"`
 	// Annotations is an unstructured key value map that may be set by external tools to store and retrieve arbitrary metadata.
 	Annotations map[string]string `json:"annotations,omitempty"`
-
+	// HypervisorPath is the absolute path to the hypervisor binary
+	// for VM-based runtimes.
+	HypervisorPath string `json:"hypervisorPath"`
```

Comment:
These settings should be optional and probably don't belong in the top level struct.